site/static/examples/admin/single-clusterqueue-setup.yaml

```diff
@@ -2,6 +2,11 @@ apiVersion: kueue.x-k8s.io/v1beta1
 kind: ResourceFlavor
 metadata:
   name: "default-flavor"
+spec:
+  nodeLabels:
+    key1: value
+    key2: value
+
```

Comment:
I just realized that this wasn't a good idea....
The example can no longer be used in a base cluster that doesn't have these labels.

Was this change necessary for the test?

Reply:
I think I missed to remove it after test. I can revert it.

Reply:
Fixed on #2928.